dashboard/app/controllers/api/v1/projects/public_gallery_controller.rb

```diff
@@ -5,11 +5,21 @@ def index
       expires_in 5.seconds, public: true # cache
     end
 
-    render json: ProjectsList.fetch_published_projects(
-      params[:project_type],
-      limit: params[:limit],
-      published_before: params[:published_before]
-    )
+    if params[:showFeatured] == "1"
+      render json: ProjectsList.fetch_published_projects(
+        params[:project_type],
+        limit: params[:limit],
+        published_before: params[:published_before],
+        append_featured: true
```

Comment:
if the only difference is one param, you could get rid of the if / else and just pass `append_featured: !!params[:showFeatured]`

Reply:
nit: it might be more accurate to call this `prepend_featured`, if you are in fact prepending